### PR TITLE
Updated config-bootstrapper docs

### PR DESCRIPTION
**Pull request by geekodour**
cc @stevekuznetsov 

Signed-off-by: Hrishikesh Barman <hrishikeshbman@gmail.com>

### CODE REVIEW

**Comment by geekodour** [comment]
I created the cla/linuxfoundation account with github login, don't know why it is still failing.

**Comment by matthyx** [comment]
> If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits.

**Comment by geekodour** [comment]
I signed it

File: prow/cmd/config-bootstrapper/README.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ config-updater Prow plugin.
 When a set of configurations do not exist (for example, on a clean redeployment or in a disaster
 recovery situation), the config-updater plugin is not useful as it can only upload incremental
 updates. This tool is meant to be used in those situations to set up the config to the correct
-base state and hand off ownership to the plugin for updates.
\ No newline at end of file
+base state and hand off ownership to the plugin for updates.
+
+`config-bootstrapper` uses the [`updateconfig`](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/updateconfig) package to update the configmaps and uses configurations from `plugins.yaml` and `config.yaml` that are specified as command line arguments to it.
```
Comment (stevekuznetsov) [inline prow/cmd/config-bootstrapper/README.md]: I think for a user who just wants their config to work, specifying which package in code is used to run the tool is a little too low-level. Maybe we could write:

> Provide the `config-bootstrapper` with the latest state of the Prow configuration (`plugins.yaml`, `config.yaml`, any job configuration files) to boot-strap with the latest configuration.

Comment (geekodour) [reply prow/cmd/config-bootstrapper/README.md]: I added the changes with a usage example, let me know if I need to make any other changes.

**Comment by matthyx** [comment]
/lgtm
@stevekuznetsov ?

**Pull request merged**